tests/highlights.test.js

```diff
@@ -21,8 +21,8 @@ describe('Like Comments', () => {
       });
   });
 
-  describe('/Create a comment on a highlighted text in article', () => {
-    it('should comment on highlighted text and return 201', (done) => {
```

Comment:
I think there is nothing wrong with this description. I don't see the point of changing it.